soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -7265,11 +7265,41 @@ s32 func_8083FD78(Player* this, f32* arg1, s16* arg2, PlayState* play) {
             *arg2 = this->actor.shape.rot.y;
         }
 
-        if (this->unk_664 != NULL) {
-            func_8083DB98(this, 1);
+        // #region SOH [Enhancement]
+        if (CVarGetInteger("gRightStickAiming", 0) != 0 || CVarGetInteger("gInvertZAimingYAxis", 1) == 0) {
```

Comment:
Not a blocker, but in general I've been moving bool cvar checks away from `!=` and `==` 0, just for future reference.

Reply:
Rewrote these and the one below, as well.